data/map.txt

```diff
@@ -25856,7 +25856,7 @@ planet Dune
 planet "Dusk Companion"
 	attributes kimek oil
 	landscape land/mountain9
-	description `Dusk Companion's moon is so large and orbits so close to the planet that the tidal forces even affect the planet's plate tectonics. The skies here are frequently blanketed in a volcanic haze. This world is rich in oil deposits, a relic of a past when more life flourished here, but drilling for oil is costly because the shifting ground can damage drills or seal up previously productive bore-holes.`
+	description `Dusk Companion's moon is so large and orbits so close to the planet that the tidal forces even affect the planet's plate tectonics. The skies here are frequently blanketed in a volcanic haze. This world is rich in oil deposits, a relic of a past when more life flourished here, but drilling for oil is costly because the shifting ground can damage drills or seal up previously productive bore-holes. For this reason the oil is exported off-world for processing into plastics, instead of having the manufacturing performed locally.`
```

Comment:
Why would processing it elsewhere help, when the problem is with producing it?

Reply:
It's meant to be a flag for why plastics are so expensive on that world. Plastics are much easier to transport than crude oil, and so everywhere else that oil is in the planet attributes you find plastics are fairly cheap. This world stands out as being different. Plastic processing is significantly more delicate than oil extraction, so it's is not something you want to do with a lot of tectonic activity. So this says, "Hey yeah the oil is here, but god it's a nightmare so once it's out of the ground we just send it away because this place sucks."

Reply:
In that case, maybe clarify it a bit more that the tectonic activity is the issue? as it stands, `For this reason` appears to refer to the fact that `drilling for oil is costly`.

e.g.
```
For the same reason the oil is exported...
```

```
Since this also makes processing difficult, the oil is exported...
```

Reply:
I'm happy with that wording, yeah.